src/main/java/net/minecraftforge/client/model/geometry/UnbakedGeometryHelper.java

```diff
@@ -98,17 +99,28 @@ public static BakedModel bake(BlockModel blockModel, ModelBakery modelBakery, Bl
         if (customModel != null)
             return customModel.bake(blockModel.customData, modelBakery, spriteGetter, modelState, blockModel.getOverrides(modelBakery, owner, spriteGetter), modelLocation);
 
-        Transformation rootTransform = blockModel.customData.getRootTransform();
-        if (!rootTransform.isIdentity())
-            modelState = new SimpleModelState(modelState.getRotation().compose(rootTransform), modelState.isUvLocked());
-
         // Handle vanilla item models here, since vanilla has a shortcut for them
         if (blockModel.getRootModel() == ModelBakery.GENERATION_MARKER)
             return ITEM_MODEL_GENERATOR.generateBlockModel(spriteGetter, blockModel).bake(modelBakery, blockModel, spriteGetter, modelState, modelLocation, guiLight3d);
 
-        var renderTypeHint = blockModel.customData.getRenderTypeHint();
-        var renderTypes = renderTypeHint != null ? blockModel.customData.getRenderType(renderTypeHint) : RenderTypeGroup.EMPTY;
-        return blockModel.bakeVanilla(modelBakery, owner, spriteGetter, modelState, modelLocation, guiLight3d, renderTypes);
+        return bakeVanilla(blockModel, modelBakery, owner, spriteGetter, modelState, modelLocation);
+    }
+
+    /**
+     * Helper for baking vanilla {@link BlockModel} instances.
+     *
+     * @deprecated Merge into the method above in 1.20 once the call from {@link BlockModel} is gone.
+     */
+    @ApiStatus.Internal
+    @Deprecated(forRemoval = true, since = "1.19.2")
+    public static BakedModel bakeVanilla(BlockModel blockModel, ModelBakery modelBakery, BlockModel owner, Function<Material, TextureAtlasSprite> spriteGetter, ModelState modelState, ResourceLocation modelLocation) {
```

Comment:
bracket son newlines

Reply:
Completely missed this since it was copy-pasted from another place. Fixed!